app/views/annotations/update_annotation.js.erb

```diff
@@ -9,6 +9,6 @@ document.getElementById('annotation_summary_list').innerHTML =
                                   locals: { annotation_category: @annotation_text.annotation_category }) %>';
 <% end %>
 
-update_annotation_text('<%= @annotation_text.id %>', '<%= escape_javascript(simple_format(@annotation_text.content)) %>');
+update_annotation_text('<%= @annotation_text.id %>', marked('<%= escape_javascript(simple_format(@annotation_text.content)) %>'));
```

Comment:
@finnergizer This is the only one that catches my eye. There's an inner call to `simple_format`, why not use `markdown` here?